### PR TITLE
policy with a long name not working properly

### PR DESCRIPTION
**Pull request by ChunxiAlexLuo**
https://github.com/open-cluster-management/backlog/issues/10215

### CODE REVIEW

**Review by dhaiducek — APPROVED** [review]

**Comment by dhaiducek** [comment]
FYI: The last two E2E tests here will need to be updated also:
https://github.com/open-cluster-management/grc-ui/blob/ce3c9411eb30ea87bc8e0c6cf24dd0e1c4bbc054/tests/e2e/all-policy-page.test.js#L57-L74

**Comment by dhaiducek** [comment]
This seems reasonable to me to add to this fix:
>It seems reasonable to check the length of <policyNamespace>.<policyName> and warn the user eventually, because there are namespaces having more than 10 characters.
https://github.com/open-cluster-management/backlog/issues/10215#issuecomment-796706119

**Comment by abhiNamdhari** [comment]
@ChunxiAlexLuo Can you please update this error message as well https://github.com/open-cluster-management/grc-ui/blob/main/tests/cypress/config/InvalidYamlTests/invalidYamlErrors.yaml#L2?

File: nls/platform.properties [hunk 1]
```diff
@@ -93,7 +93,7 @@ error.create.policy = Create error:
 error.policy.nameFormat.short = Invalid name: should only have lowercase alphanumeric characters, '-', or '.' and not begin or end with punctuation
 error.policy.nameFormat.hint = Invalid name due to Kubernetes naming restriction.
 error.policy.nameFormat = The name must meet the following requirements:
-error.policy.nameFormat.Rule1 = \u2022 contain no more than 253 characters
+error.policy.nameFormat.Rule1 = \u2022 the combined length of namespace and policy name should not exceed than 63 characters
```
Comment (dhaiducek) [inline nls/platform.properties:96]: Remove 'than'

**Pull request merged**